src/server.js

```diff
@@ -440,6 +434,8 @@ function initializeClient(socket, client, token, lastMessage) {
 			networks: networks,
 			token: tokenToSend,
 		});
+
+		console.timeEnd(`${client.name} - time to emit to socket.io`);
```

Comment:
Debugging leftovers.

Reply:
I left it in to measure perf.

Reply:
Right but we don't want to merge with this, right?

Reply:
I will remove it later before merging.